redis_enterprise/assets/dashboards/redis-enterprise-cluster-dashboard.json

```diff
@@ -0,0 +1,455 @@
+{
+  "title": "Redis Enterprise Cluster Dashboard",
+  "description": "",
```

Comment:
Dashboard is missing a description, see [best practices guidelines](https://docs.datadoghq.com/developers/integrations/create-an-integration-dashboard?tab=ui#follow-dashboard-best-practices).